src/index.html

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big">
+      <label for="search1" class="search-Label search">
+        <input id="search1" type="text"
+        data-qa="keypress" placeholder="Try &quot;Los Angeles&quot;"
+        class="input big">
+      </label>
+    </form>
+    <form data-qa="small">
+      <label for="search2" class="search-Label small-search search">
+        <input id="search2" type="text"
+        placeholder="Try &quot;Los Angeles&quot;" class="input input-small">
```

Comment:
```suggestion
        placeholder="Try &quot;Los Angeles&quot;" class="input input--small">
```
Модификатор пишется через два дефиса